app/javascript/components/public_health/PublicHealthHeader.js

```diff
@@ -127,7 +127,7 @@ class PublicHealthHeader extends React.Component {
           {this.state.importType === 'saf' && (
             <div className="mb-3">
               <a href="https://github.com/SaraAlert/SaraAlert/blob/master/public/Sara%20Alert%20Import%20Format.xlsx?raw=true">Download formatting guidance</a>{' '}
-              (Updated 2/9/2021)
+              (Updated 3/9/2021)
```

Comment:
Needs to be updated to 3/23/2021. Same with the dates in the import template.

Reply:
fixed: https://github.com/SaraAlert/SaraAlert/pull/689/commits/39ae0cc24006c7ef696115836d7421f42e95429c

Reply:
Looks like you updated the import template (thanks!), however this line will also need to be updated

Reply:
sorry, thought I updated it when I didn't, should be fixed now